app/assets/index.html

```diff
@@ -20,5 +20,6 @@
 </div>
 
 <script src="scripts/main.js"></script>
+<script>require('app/scripts/main');</script>
```

Comment:
I see. I ultimately want to make this app CSP-friendly, but I don't want to sacrifice performance, so I'm leaning towards the `modules.js` alternative. What would the contents of this file look like, and why are you deeming it "config-unfriendly"?